# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../corona/corona-1.0.2-gcc43.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- corona-1.0.2/src/OpenTGA.cpp.orig	2011-04-17 19:09:34.696077822 +0200
+++ corona-1.0.2/src/OpenTGA.cpp	2011-04-17 19:10:45.228077811 +0200
@@ -1,3 +1,4 @@
+#include <cstring> 	// memcpy
 #include <algorithm>
 #include "Debug.h"
 #include "Open.h"
--- corona-1.0.2/src/Convert.cpp.orig	2011-04-17 19:11:25.048077816 +0200
+++ corona-1.0.2/src/Convert.cpp	2011-04-17 19:11:52.364070520 +0200
@@ -3,6 +3,7 @@
  * @todo  allow conversions from direct color images to
  *        palettized images
  */
+#include <cstring>	// memcpy
 #include <map>
 #include <utility>
 #include "corona.h"
